### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kicotiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,83 @@
...
 * Database creation
+- Devin Altobello - [GitHub](https://github.com/daltobello) || [LinkedIn](https://www.linkedin.com/in/devin-altobello-2100036b/)
+- Jen Ziel - [GitHub](https://github.com/jenziel) || [LinkedIn](https://www.linkedin.com/in/jen-ziel400/)
+- Marisa Wyatt - [GitHub](https://github.com/Marisa5280) || [LinkedIn](https://www.linkedin.com/in/marisarwyatt/)
 
-* Database initialization
+## Technologies
+Primary:
+- Ruby 3.2.2
+- Rails 7.0.8
```
Comment (Antoine-Aube) [inline README.md:30]: Add docker here?

**Comment by Antoine-Aube** [comment]
looks lickity split to me. Thank you for putting this together.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,84 @@
...
+### Build with Docker
+- Ensure [Docker](https://www.docker.com/) is installed locally
+- Create a docker image: `docker build -t . experience_exchange_images`
+- Run the app via the docker image: `docker run -p 5000:5000 experience_exchange_images` (note your 5000 port will need to be unbound, otherwise specify a separate port)
+
+### Third Party APIs
+Experience Exchange uses two third party APIs - [Geoapify](https://apidocs.geoapify.com/docs/geocoding/forward-geocoding/#about) for geocoding, and [Unsplash](https://unsplash.com/documentation) for randomized user images. Only Unsplash needs to be setup in this microservice repo for retrieval of an image for a user's profile.
+
+Follow the Unsplash documentation to get an API key, and place it in the rails credentials file as follows:
+- `EDITOR="code --wait" rails credentials:edit`
```
Comment (ethanb1145) [inline README.md:54]: Really liked how you showed how to setup an api key for this repo!

**Review by ethanb1145 — APPROVED** [review]
Looks neat, easy to follow, and makes it simple on how to setup! Thank you!

**Pull request merged**